sdk/core/Azure.Core.Experimental/api/Azure.Core.Experimental.netstandard2.0.cs

```diff
@@ -99,58 +124,87 @@ public sealed partial class GeoJsonConverter : System.Text.Json.Serialization.Js
         public override Azure.Core.GeoJson.GeoObject Read(ref System.Text.Json.Utf8JsonReader reader, System.Type typeToConvert, System.Text.Json.JsonSerializerOptions options) { throw null; }
         public override void Write(System.Text.Json.Utf8JsonWriter writer, Azure.Core.GeoJson.GeoObject value, System.Text.Json.JsonSerializerOptions options) { }
     }
-    public sealed partial class GeoLine : Azure.Core.GeoJson.GeoObject
+    public sealed partial class GeoLinearRing
```

Comment:
Is this for a polygon? The name strays way too far from the standards and isn't obvious. Polygon is a general term. At a glance, no idea what a "linear ring" is. I had to double check against GeoJSON standards to check what this might be based on the data structure.

Reply:
It's somewhat from the standard: https://tools.ietf.org/html/rfc7946#section-3.1.6

It's not exactly one of the defined geometry types but they do use the term